Buildings/ThermalZones/Detailed/Validation/BESTEST/Cases6xx/Case600.mo

```diff
@@ -73,6 +73,47 @@ model Case600 "Case 600FF, but with dual-setpoint for heating and cooling"
   Buildings.Controls.OBC.CDL.Reals.MovingAverage PCoo(delta=3600)
     "Hourly averaged cooling power"
     annotation (Placement(transformation(extent={{-20,-8},{-12,0}})));
+  Modelica.Blocks.Sources.RealExpression hGloHor(
+    y=weaDat.weaBus.HGloHor)
+    "Global horizontal solar irradiance"
+    annotation (Placement(transformation(extent={{40,20},{60,40}})));
+  Modelica.Blocks.Continuous.Integrator gloHor(
+    k=1,
+    initType=Modelica.Blocks.Types.Init.InitialState,
+    y_start=0) "Annual global horizontal solar irradiance"
+    annotation (Placement(transformation(extent={{74,26},{82,34}})));
+  replaceable Modelica.Blocks.Sources.RealExpression hGloSou(
+    y=roo.conExtWinRad[1].HDir + roo.conExtWinRad[1].HDif)
+    "South total solar irradiance"
+    annotation (Placement(transformation(extent={{40,0},{60,20}})));
+  Modelica.Blocks.Continuous.Integrator gloSou(
+    k=1,
+    initType=Modelica.Blocks.Types.Init.InitialState,
+    y_start=0) "Annual south total solar irradiance"
```

Comment:
This should be "global" not "total" in the comment. Also above for `hGloSou`

Reply:
changed.